crates/rocksdb/src/lib.rs

```diff
@@ -551,7 +555,7 @@ impl RocksDb {
                     e
                 );
             } else {
-                info!(
+                debug!(
```

Comment:
Can we revert this irrelevant change?

Reply:
I thought it was useful to silence the startup message:

```
2025-06-23T15:36:53.048899Z  INFO shutdown: restate_rocksdb: 2 column families flushed in 27.625µs db=local-metadata-store db=local-metadata-store
```

But yes, sure!